src/Tizen.NUI/src/public/ViewProperty/BackgroundExtraData.cs

```diff
@@ -32,6 +32,11 @@ internal BackgroundExtraData()
         internal BackgroundExtraData(BackgroundExtraData other)
         {
             BackgroundImageBorder = other.BackgroundImageBorder;
+
+            // Deprecated in API13
+            CornerRadius = other.CornerRadius;
```

Comment:
Also, Remove this Obsolete for these properties.

Reply:
I think we don't need to remove Obsolete here. 
Users may set/get view properties, not BackgroundExtraData. Duplication at setter is just for the animation.